flink/v2.1/flink/src/main/java/org/apache/iceberg/flink/data/FlinkValueReaders.java

```diff
@@ -191,10 +191,10 @@ private static class TimestampNanosReader implements ValueReader<TimestampData>
 
     @Override
     public TimestampData read(Decoder decoder, Object reuse) throws IOException {
-      long nanos = decoder.readLong();
-      long mills = Math.floorDiv(nanos, 1_000_000);
-      int leftover = Math.floorMod(nanos, 1_000_000);
-      return TimestampData.fromEpochMillis(mills, leftover);
+      long storedValue = decoder.readLong();
+      long mills = Math.floorDiv(storedValue, 1_000_000L);
+      int leftoverNanos = (int) Math.floorMod(storedValue, 1_000_000L);
+      return TimestampData.fromEpochMillis(mills, leftoverNanos);
```

Comment:
Any relevant change here?
For me this seems like we renamed some of the variables, but nothing has changed